### PR TITLE
BZ2013890: Missing OLM metrics

### PR DESCRIPTION
**Pull request by michaelryanpeter**
Version(s): 4.9+

Issue: [BZ-2013890](https://bugzilla.redhat.com/show_bug.cgi?id=2013890) and adding the additional metric `olm_resolution_duration_seconds` from [OCPBUGS-5309](https://issues.redhat.com/browse/OCPBUGS-5309)
UPDATE: BZ migrated to Jira in ticket: [OCPBUGS-9002](https://issues.redhat.com/browse/OCPBUGS-9002)

Link to docs preview: [Understanding Operators > OLM > Metrics](https://56741--docspreview.netlify.app/openshift-enterprise/latest/operators/understanding/olm/olm-understanding-metrics.html#olm-metrics_olm-understanding-metrics)

QE review:
- [x] QE has approved this change.

Additional information:

The BZ mentioned a known issue in addition to the missing metrics. I split the known issue out into a separate ticket: [OCPBUGS-4663](https://issues.redhat.com/browse/OCPBUGS-4663)

### CODE REVIEW

File: modules/olm-metrics.adoc [hunk 1]
```diff
@@ -30,6 +33,9 @@ Operator Lifecycle Manager (OLM) exposes certain OLM-specific resources for use
 |`install_plan_count`
 |Number of install plans.
 
+|`installplan_warnings_total
```
Comment (dtfranz) [inline modules/olm-metrics.adoc:36]: Missing closing ` mark?

Comment (michaelryanpeter) [reply modules/olm-metrics.adoc]: Fixed. Thank you!

**Review by dtfranz — APPROVED** [review]
/approve

**Comment by michaelryanpeter** [comment]
I updated the PR to include the `olm_resolution_duration_seconds` metric that was mentoined in [OCPBUGS-5309](https://issues.redhat.com/browse/OCPBUGS-5309).

**Comment by jianzhangbjz** [comment]
Thanks!
/lgtm

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Pull request merged**